tdrs-backend/tdpservice/reports/models.py

```diff
@@ -61,31 +94,45 @@ class Meta:
 
         constraints = [
             models.UniqueConstraint(
-                fields=("section", "version", "quarter", "year", "stt"),
-                name="constraint_name",
+                fields=("version", "quarter", "year", "stt"),
+                name="unique_reports_reportfile_fields",
             )
         ]
 
+    # Override FileRecord fields
+    extension = models.CharField(max_length=8, default="zip")
+
+    # Model Fields
     created_at = models.DateTimeField(auto_now_add=True)
     quarter = models.CharField(
         max_length=16, blank=False, null=False, choices=Quarter.choices
     )
     year = models.IntegerField()
-    section = models.CharField(
-        max_length=32, blank=False, null=False, choices=Section.choices
-    )
 
     version = models.IntegerField()
```

Comment:
curious about how the report version is meant to be used. does a report correspond to a specific datafile/version? and should the datafile be linked via fk instead?